src/views/pages/partials/product/options.twig

```diff
@@ -60,7 +60,7 @@
 {% endif %}
 
 
-{% if product.can_add_note or product.can_upload_file %}
+{% if not isCartOption and (product.can_add_note or product.can_upload_file) %}
```

Comment:
no need for this, condition, the `product.can_add_note` and `product.can_upload_file` should be false

Reply:
Because without it the upload file and not items got displayed, I asked Abu Baker if he is sure that the values `product.can_add_note` and `product.can_upload_file` will be always false, he told me he is not sure, thats why I made this extra condition to be sure that they will not be displayed in case of cart options. **_should I keep it or delete the condition?_**

Reply:
No need, remove them, will be handled from backend